tests/react/name-input.tests.jsx

```diff
@@ -0,0 +1,70 @@
+var React = require('react');
+var TestUtils = require('react/lib/ReactTestUtils');
+var should = require('should');
+var IntlContext = { messages: {'first_name': 'first name', 'last_name': 'last name'} };
+var stubContext = require('react-test-context');
+var Name = require('../../components/name-input.jsx');
+
+describe('input components', function () {
+  describe('name-input.jsx', function () {
+
+    it('should require a field props', function () {
+      should.throws(() => {
+        TestUtils.renderIntoDocument(React.createElement(stubContext(Name, IntlContext)));
+      });
+    });
+
+    it('should throw when passing wrong prop types', function () {
+      should.throws(() => {
+        TestUtils.renderIntoDocument(React.createElement(stubContext(Name, IntlContext), {name:'ali', onChange: 'lol'}));
+      });
+    });
+
+    it('should not throw when passing all props', function () {
+      should.doesNotThrow(() => {
+        var NameInput = stubContext(Name, IntlContext);
+        TestUtils.renderIntoDocument(<NameInput name='ali' onChange={function(){}} />);
+      });
+    });
+    it('.validate() should return false when first name and last name field are empty', function () {
+      should.doesNotThrow(() => {
+        var NameInput = stubContext(Name, IntlContext);
+        var el = TestUtils.renderIntoDocument(<NameInput name='ali' onChange={function(){}} />);
```

Comment:
I just want to pass in some random string there, but if you want to change the prop name `name` maybe we can change it in another patch or something?